services/bundlejs/bundlejs-package.tester.js

```diff
@@ -0,0 +1,33 @@
+import { isFileSize } from '../test-validators.js'
+import { createServiceTester } from '../tester.js'
+export const t = await createServiceTester()
```

Comment:
Just for clarity, is the strategy we're relying on for the tests here just picking really popular packages that are likely to be already cached at any given time?

Reply:
Bundlejs now [permanently caches the result](https://github.com/badges/shields/pull/9055#discussion_r1292840083).